pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+'''
+    File:               DDM.py
+    Author:             Nikita Vodyagin, QuTech
+    Purpose:            control of Qutech DDM
+    Prerequisites:
+    Usage:
+    Bugs:
+'''
+
+
+from .SCPIddm import SCPIddm
+import numpy as np
+import struct
+from qcodes import validators as vals
+
+
+class DDMq(SCPIddm):
+
+    # def __init__(self, logging=True, simMode=False, paranoid=False):
+    def __init__(self, name, address, port, **kwargs):
+        super().__init__(name, address, port, **kwargs)
+        # AWG properties (to be overloaded by derived class)
+        self.device_descriptor = type('', (), {})()
+        self.device_descriptor.model = 'DDM'
+        self.device_descriptor.numChannels = 2
+        # self.numDacBits = 0
+        # self.numMarkersPerChannel = 0
+        # self.numMarkers = 0
+        self.add_parameters()
+        self.connect_message()
+        # SCPIdriver.__init__(self, logging, simMode, paranoid)
+
+    def add_parameters(self):
+        #######################################################################
+        # DDM specific
+        #######################################################################
+
+        for i in range(self.device_descriptor.numChannels//2):
+            ch_pair = 2*i+1
+            snavg_cmd = 'qutech:inputavg{}:naverages'.format(ch_pair)
+            self.add_parameter('ch_pair{}_inavg_Navg'.format(ch_pair),
+                               units='#',
+                               label=('number of averages' +
```

Comment:
This label will have a missing space between number of averages and ch_pair. 

Also, let's start Labels with capital letters.